Resources/Storage/StorageElement.py

```diff
@@ -131,7 +131,7 @@ def __init__( self, name, plugins = None, vo = None ):
         return
       self.vo = result['Value']
     self.opHelper = Operations( vo = self.vo )
-    
+
```

Comment:
In the options: PyDev -> Editor -> Code Style -> Code Formatter everything is checked there in my case.